packages/studio-be/src/migrations/v12_5_0-1577776334-text-markdown.ts

```diff
@@ -24,44 +24,37 @@ const migration: Migration = {
 
           if (!(markdownKey in formData)) {
             formData[markdownKey] = true
-            hasChanged = true
+            hasChanges = true
           }
         }
       }
     }
 
-    const migrateBotTextContent = async (botId: string) => {
-      const bpfs = ghostService.forBot(botId)
-      const entFiles = await bpfs.directoryListing(ELEMENTS_DIR, '*.json')
+    const bpfs = ghostService.forBot(botId)
+    const entFiles = await bpfs.directoryListing(ELEMENTS_DIR, '*.json')
 
-      for (const fileName of entFiles) {
-        try {
-          const contentElements = await bpfs.readFileAsObject<sdk.ContentElement[]>(ELEMENTS_DIR, fileName)
-          contentElements.forEach(element => updateFormData(element))
+    for (const fileName of entFiles) {
+      try {
+        const contentElements = await bpfs.readFileAsObject<sdk.ContentElement[]>(ELEMENTS_DIR, fileName, {
+          noCache: true
```

Comment:
When we read the file without altering anything, then we don't need the noCache. The problem is when we mutate the content, if we keep it in cache, then the next run (not the dry one) will not see that theres a change to make

Reply:
I think we should never mutate the content when in dry mode. Wouldn't that solve the issue instead of having to be aware of the noCache property and its use?